wandb/sdk/wandb_settings.py

```diff
@@ -375,6 +375,8 @@ class Settings:
     _jupyter_path: str
     _jupyter_root: str
     _kaggle: bool
+    _live_policy_rate_limit: int
+    _live_policy_wait_time: Optional[int]
```

Comment:
```suggestion
    _live_policy_wait_time: int
```
this should never be optional as it's used to generate the dynamic type validator (Optional would pass anything). All settings default to None if you don't explicitly specify otherwise.